From 30d78ba5ce7fd2aabae174bcac106d6b202c6d31 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Maty=C3=A1=C5=A1ek?= <martin.matyasek@gmail.com>
Date: Thu, 27 Jun 2024 11:58:57 +0200
Subject: [PATCH] fix(completions): fix missing line breaks in _eza

Before this change the shell would report a parsing error, which was due
to missing line breaks after the newly introduced symlinks flags. This
commit fixes that issue.
---
 completions/zsh/_eza | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/completions/zsh/_eza b/completions/zsh/_eza
index f672e0a59..67cf0284d 100644
--- a/completions/zsh/_eza
+++ b/completions/zsh/_eza
@@ -33,8 +33,8 @@ __eza() {
         {-A,--almost-all}"[Equivalent to --all; included for compatibility with \'ls -A\']" \
         {-d,--list-dirs}"[List directories like regular files]" \
         {-D,--only-dirs}"[List only directories]" \
-        --no-symlinks"[Do not show symbolic links]"
-        --show-symlinks"[Explictly show symbolic links: for use with '--only-dirs'| '--only-files']"
+        --no-symlinks"[Do not show symbolic links]" \
+        --show-symlinks"[Explictly show symbolic links: for use with '--only-dirs'| '--only-files']" \
         {-f,--only-files}"[List only files]" \
         {-L,--level}"+[Limit the depth of recursion]" \
         {-w,--width}"+[Limits column output of grid, 0 implies auto-width]" \
